src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,37 @@
+package duke.command;
+
+import duke.DukeException;
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+import duke.task.Task;
+
+/**
+ * Represents a command to add a task.
+ */
+public class AddCommand extends Command {
+    /** The task to be added. */
+    private Task task;
+
+    /**
+     * Creates a new add command with the specified task to be added.
+     * @param task The task to be added.
+     */
```

Comment:
This comment is missing the empty line between description and parameter section as stated in the coding convention page. Happens a few times in this class, so I'll just comment once here!

Reply:
Noted!